www/types/sudo-prompt.d.ts

```diff
@@ -16,13 +16,11 @@
 // https://www.npmjs.com/package/sudo-prompt
 
 declare module 'sudo-prompt' {
-
   export interface SudoPromptOptions {
-    name?: string,
-    icns?: string
+    name?: string, icns?: string
   }
 
   export function exec(
       command: string, options?: SudoPromptOptions,
-      callback?: (error: Error, stdout: string|Buffer, stderr: string|Buffer) => void): void;
+      callback?: (error: string, stdout: string|Buffer, stderr: string|Buffer) => void): void;
```

Comment:
I believe this is an Error object from looking at the [source of sudo-prompt](https://github.com/jorangreef/sudo-prompt/blob/master/index.js).

Reply:
I thought that too at first but in practice it's always a string, whether the user refuses permissions or the command fails. I added a comment to the `.d.ts` to clarify this.